habitat/sims/habitat_simulator/action_spaces.py

```diff
@@ -0,0 +1,41 @@
+from enum import Enum
+
+import attr
+
+import habitat_sim
+from habitat.core.registry import registry
+from habitat.core.simulator import (
+    ActionSpaceConfiguration,
+    Config,
+    SimulatorActions,
+)
+
+
+@registry.register_action_space_configuration(name="v1")
```

Comment:
Can we register as v1 the one without look up/look down to be compatible with Habitat whitepaper? Our checkpoints are not loading because of the number of actions as well.

Reply:
Done.  I called the whitepaper one v0